docs/user-guide/service-accounts.md

```diff
@@ -111,6 +111,8 @@ secrets/build-robot-secret
 
 Now you can confirm that the newly built secret is populated with an API token for the "build-robot" service account.
 
+Note that a secret of type `kubernetes.io/service-account-token` that references a non-existent service account will be cleaned up by the service account token controller.
```

Comment:
How about: 

Any tokens for non-existent service accounts will be cleaned up by the token controller.